### ISSUE TITLE
On platform chat

### ISSUE THREAD

**Issue by Earlo** [hackjunction/hackplatform#51]
Do we need it? Yes?

**Comment by lappalj4** [comment]
What's the use case / who's chatting with who? 

Without knowing more, I would be a bit weary of saying this is "high priority" - there's still quite a few pretty core features (e.g. project reviewing and displaying rankings) that have quite a lot of room for improvement imo. 

But, feel free to convince me 😎

**Comment by pholm** [comment]
So, the idea here is to allow participants to chat with partner companies in the recruitment tool – quite the same concept as in Slush matchmaking tool (chat happens in-app, but notifications are sent via email from each message). We found this as a key feature to better support recruitment actions around events. 

I agree that this is not as high priority as the ones you have mentioned. So lets create corresponding issues for all of those and add them to the GitHub project :)

**Comment by lappalj4** [comment]
Ok yeah makes sense! Yeah the current way we've done it certainly isn't the best experience for partners. I'd say this is certainly doable once we get the partner profiles architecture in place

**Comment by pholm** [comment]
Yes, that is definitely to-do before this. I think @parkkot has some ideas how the partner side of things should work out (not part of this issue tho).
